.github/workflows/integration-tests-command.yml

```
@@ -152,20 +152,14 @@ jobs:
       - name: Download the rts build artifact
         uses: actions/download-artifact@v2
         with:
-          name: rts-build
+          name: rts-dist
           path: app/rts/rts-dist.tar
           
       # Untar the bundles
       - name: Untar the rts bundles
         run: |
           tar -xvf rts-dist.tar dist
```

I think this command needs to be the following:
```suggestion
          tar -xvf app/rts/rts-dist.tar -C app/rts/
```
This is because we are downloading the tar in `app/rts/` folder while running the untar command from the root folder. Hence, we should give the full path name & also the directory into which we want the contents to lie.

---

added cd into app/rts

---

made the chages